Util/BuildTools/BuildUtilsDocker.sh

```diff
@@ -11,13 +11,15 @@ if [ -f "${FBX2OBJ_DIST}/FBX2OBJ" ]; then
   log "FBX SDK already installed."
   exit
 fi
-
 LIB_NAME=fbx202001_fbxsdk_linux
 FBXSDK_URL=https://www.autodesk.com/content/dam/autodesk/www/adn/fbx/2020-0-1/${LIB_NAME}.tar.gz
 
 if [ ! -d "${FBX2OBJ_DEP_FOLDER}" ]; then
   log "Downloading FBX SDK..."
-  wget -c "${FBXSDK_URL}" -P "${CARLA_DOCKER_UTILS_FOLDER}"
+  wget --user-agent="Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:94.0) Gecko/20100101 Firefox/94.0" \
+      -c "${FBXSDK_URL}" -P "${CARLA_DOCKER_UTILS_FOLDER}"
+  # info: need to fake accessing from a Firefox browser, so the server doesn't return 403 forbidden
+  # see https://github.com/carla-simulator/carla/issues/4862
```

Comment:
Is this a change that *should* be included in the main repository and you fixed it on our fork according to the issue 4862 mentioned in your comment? Just for my understanding.

Reply:
Yes exactly. The last Carla release 0.9.13 on DockerHub is now 9 months ago and noone seems to care that the Docker build ist broken 😅😅😅

Reply:
I just checked the active PRs in the repo and this https://github.com/carla-simulator/carla/pull/5631 seems to fix the problem. Is this the same/better/worse solution compared to your approach?

Reply:
I'm using exactly the same command. And yes, it seems to fix the problem (although I don't understand why they need to build the Docker tools inside a Docker container 🤔)